mne_bids/mne_bids.py

```diff
@@ -100,8 +100,13 @@ def _channels_tsv(raw, fname, verbose):
     # get the manufacturer from the file in the Raw object
     manufacturer = None
     if hasattr(raw, 'filenames'):
-        _, ext = _parse_ext(raw.filenames[0], verbose=verbose)
-        manufacturer = MANUFACTURERS[ext]
+        # NOTE: Hack for EEGLAB bug in MNE-Python 0.16; fixed in MNE-Python
+        # 0.17, ... remove the hack after upgrading dependencies in MNE-BIDS
+        if raw.filenames[0] is None:  # hack
+            ext = '.set'  # hack
+        else:
+            _, ext = _parse_ext(raw.filenames[0], verbose=verbose)
+            manufacturer = MANUFACTURERS[ext]
 
```

Comment:
This was fixed here: https://github.com/mne-tools/mne-python/pull/5504

currently only available in MNE-Python 0.17